tests_recording/test_api.py

```diff
@@ -59,9 +58,9 @@ def test_comment_in_spec(self):
         """
         change specfile little bit to have there some change, do not increase version
         """
-        specfile_location = os.path.join(self.lp.working_dir, "python-ogr.spec")
+        specfile_location = Path(self.lp_working_dir, "python-ogr.spec")
```

Comment:
The same 2 questions as above (i.e. is it `self.lp.working_dir`? and why it's `str`?)